### PR TITLE
fix(r-select): update select component, support custom label and value

### PR DESCRIPTION
**Pull request by qqcky**
While integrating `r-select` component, there were found some issues. 
- support custom labels and values
- 

### Check lists

- [ ] Readme file updated with actual description and example
- [ ] Added all ARIA attributes required for component
- [x] Added tests for both browser and SSR render and for all components properties
- [x] Added story with all knobs and actions

### CODE REVIEW

**Comment by ZengJunyong** [comment]
For r-select, do we have `invalid` state in story? like this r-input.
image

As we need it in Recomm. I didn't see it.
image

**Comment by ZengJunyong** [comment]
A little difference, the bottom border is hidden in recomponent

image

File: packages/recomponents/src/components/r-select/r-select.vue [hunk 1]
```diff
@@ -530,6 +534,15 @@
             hasLabel() {
                 return (this.label || '').trim() !== '';
             },
+            computedClasses() {
+                return this.isInvalid ? 'is-error' : '';
+            },
```
Comment (followyouhome) [inline packages/recomponents/src/components/r-select/r-select.vue]: we have `classes` in other components for classes for root element, let's try to keep naming consistent

Comment (ZengJunyong) [reply packages/recomponents/src/components/r-select/r-select.vue]: https://github.com/Rebilly/rebilly-recomponents/pull/56/commits/04160e69b96bb62a2ca0d8e638be4b2c75b32176

**Comment by ZengJunyong** [comment]
For this PR, looks good. so I approve it.
We'll use r-select in Recomm, I believe we will find more issues at that time. let's create another PR then.

**Pull request merged**